Updating with new sandbox form (#499)
cncf/toc: process/project_proposals.adoc

=== Sandbox process

Note: We have changed the sandbox proposal process to a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[new lighter-weight process for Sandbox proposals].

All exceptions (and "no" outcomes) are handled by the TOC. Possible "no" outcomes include "not at this time", and the project may be encouraged to re-apply after addressing issues. Timeframes are approximate, to set expectations.

image::sandbox-process.png[Sandbox process]
. *Project Proposal*

>    * Project proposed through a https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit[form]

Reviewer: ```suggestion
   * Project proposed through a https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit[form] which populates a *LINK TO BE ADDED*[public spreadsheet]
   * Projects may wish to notify the TOC community that they are applying, through a note in the Slack channel or on the mailing list
   * Public comments on the spreadsheet are welcome
   * CNCF staff check submissions for administrative criteria, and advise project maintainers if they don’t currently meet these criteria:
       .. Compliance with IP policy
       .. Project website meets CNCF guidelines
       .. Code of Conduct exist
       .. Contribution guidelines exist 
       .. Public roadmap exists. 
 
   * There is a two week cut-off before the TOC submission review meeting, to allow for public comment and for TOC to look at materials. Any submissions after the cut-off simply roll forward to the next review cycle.
```

Adding detail, taken from the google doc on the new Sandbox process

>    * Project proposed through a https://docs.google.com/document/d/1IlhqnmhSzs3aTPMkp_75JPYz6LD21aK1sNPrBNruDt0/edit?usp=sharing[form]

Reviewer 2: This link did not lead me to a form

Author: Resolved!

>    * Project proposed for contribution through a https://github.com/cncf/toc/pulls[Pull request] 

Reviewer 2: Please remove the pull request option as we're only doing form now. This is confusing.

Author: No, we'll need to have something to merge in at the end of the process -- we still need a pull request to merge into the repo.

Reviewer 2: ahh that makes sense

Reviewer 2: Ah that makes sense but I was not able to gather that from the bullets. Maybe numbering them would help?

Reviewer: I am surprised about this, I also thought we were just doing the form. What's the content that we need for merging into the repo? And could this be populated after the vote if it's successful, to keep the application process lightweight?

. *TOC Review*

>    * TOC reviews proposal in spreadsheet and presentation, project passes for inclusion with a simple majority vote. 

Reviewer: I think we can remove "and presentation" since that is optional

Reviewer: ```suggestion
   * TOC review Sandbox proposals in spreadsheet in a meeting every two months 
   * TOC use the following criteria:
      .. Is the project a fit for the CNCF, according to the charter and TOC principles, in the TOC’s opinion? 
      .. Is the project’s roadmap in line with the goals of the CNCF as defined in the charter and TOC principles? 
      .. Does the project appear to be on a good path to becoming well-governed and vendor-neutral? 
   * TOC votes on each submission, and projects will be accepted by a simple majority vote of the TOC. 
   * In the event of rejection, we will indicate which of these criteria the TOC did not believe were met. Projects are welcome to address feedback and submit again in future. Unless advised otherwise, projects should not submit again for six months. 
```

Adding detail on the criteria we use, taken from the Sandbox proposal google doc

>    * TOC may engage with project to ask further questions

Reviewer 2: Please add a bullet that describes the TOC vote in this list

>    * Projects may re-apply after six months

Reviewer 2: I think projects should be able to re-apply after addressing concerns if they were not accepted.

Author: There was a conversation in the proposal doc about 'wait 6 months before reapplying', and that's why that's here.

. *Governance / legal issues*
   * CNCF staff handle governance / legal issues

>    * This can happen after TOC Sandbox Sponsors step forward

Reviewer: ```suggestion
   * This can happen after the TOC vote (in other words, the TOC vote for acceptance is subject to any governance / legal issues being resolved)
```
